From 50c98190c3e82b93d79c81e29921b19f34f9ef24 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <karel.koci@nic.cz>
Date: Thu, 2 Jan 2020 14:38:01 +0100
Subject: [PATCH] target/linux/generic: enable BTRFS

Turris routers use BTRFS as root filesystem. This enables is globally
for all Turris OS builds on Linux 4.14.
---
 target/linux/generic/config-4.14 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/linux/generic/config-4.14 b/target/linux/generic/config-4.14
index b0805b1..2961b23 100644
--- a/target/linux/generic/config-4.14
+++ b/target/linux/generic/config-4.14
@@ -639,7 +639,7 @@ CONFIG_BROKEN_ON_SMP=y
 # CONFIG_BT is not set
 # CONFIG_BTRFS_ASSERT is not set
 # CONFIG_BTRFS_DEBUG is not set
-# CONFIG_BTRFS_FS is not set
+CONFIG_BTRFS_FS=y
 # CONFIG_BTRFS_FS_POSIX_ACL is not set
 # CONFIG_BTRFS_FS_RUN_SANITY_TESTS is not set
 # CONFIG_BT_ATH3K is not set
-- 
2.24.1

